app/graphql/types/invoices/object.rb

```diff
@@ -53,7 +53,7 @@ class Object < Types::BaseObject
       # NOTE(legacy): Remove with coupon before VAT refactor
       field :coupon_total_amount_cents, GraphQL::Types::BigInt, null: false, method: :coupons_amount_cents
       field :credit_note_total_amount_cents, GraphQL::Types::BigInt, null: false, method: :credit_notes_amount_cents
-      field :sub_total_vat_excluded_amount_cents, GraphQL::Types::BigInt, null: false, method: :fees_amount_cents
+      field :subtotal_before_prepaid_credits, String, null: false, method: :sub_total_vat_excluded_amount_cents
```

Comment:
`subtotal_before_prepaid_credits` method was used only in legacies invoices and probably not in the front-end.

I kept it to not break type definition for now but I'm planning to clean this type and remove all fields that are not used on the front-end side.

Regarding the method, I reused `sub_total_vat_excluded_amount_cents` because it was the used for the same purpose and I filled t with the right value in the migration `https://github.com/getlago/lago-api/pull/986/files#diff-60cf1f9f35aa85cbbadd96b65599a706207d3e987065f2940dfe758ba5ed2622R13`